presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTypeHandle.java

```diff
@@ -27,21 +27,21 @@
     private final int jdbcType;
     private final Optional<String> jdbcTypeName;
     private final int columnSize;
-    private final int decimalDigits;
+    private final Optional<Integer> decimalDigits;
     private final Optional<Integer> arrayDimensions;
 
     @JsonCreator
     public JdbcTypeHandle(
             @JsonProperty("jdbcType") int jdbcType,
             @JsonProperty("jdbcTypeName") Optional<String> jdbcTypeName,
             @JsonProperty("columnSize") int columnSize,
-            @JsonProperty("decimalDigits") int decimalDigits,
+            @JsonProperty("decimalDigits") Optional<Integer> decimalDigits,
```

Comment:
Please extract this refactoring change to separate commit. Also please describe the motivation behind this in the commit message.